e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -178,17 +178,18 @@ describe('team management', () => {
         cy.get('[data-cy=id-label]').contains(generatedTeamID);
 
         // initial state of dropdown
-        cy.get('app-team-management app-projects-dropdown #projects-selected').contains(unassigned);
+        cy.get('app-team-management app-projects-dropdown #projects-selected')
```

Comment:
There are several more places in this function (like this) where you could use the introduced constant `projectDropdown`.